src/components/ImageSettingsModal.jsx

```diff
@@ -15,38 +15,63 @@ export default class ImageSettingsModal extends Component {
       newFileName: fileName,
       sha: '',
       content: null,
+      isUpload: null,
     };
   }
 
   async componentDidMount() {
-    const { match, image: { fileName } } = this.props;
+    const { match, image, isUpload } = this.props;
```

Comment:
maybe instead of `isUpload` would it be better to call it `isPendingUpload` or `hasPendingUpload`?

Reply:
Fixed in 3ea87ca